source/_integrations/cover.mqtt.markdown

```diff
@@ -272,7 +272,7 @@ unique_id:
   description: An ID that uniquely identifies this cover. If two covers have the same unique ID, Home Assistant will raise an exception.
   required: false
   type: string
-value_template:
+state_value_template:
```

Comment:
This is not deprecated? <https://github.com/home-assistant/core/blob/dev/homeassistant/components/mqtt/cover.py>

Reply:
That's right, `state_value_template` is in fact not supported by MQTT cover.
@a-r-j-a-n can you please give some more background to why you propose this change?